src/cf-forms/src/atoms/select.less

```diff
@@ -53,7 +53,7 @@
 }
 
 
-// TODO Add modernizr to CF so this works
+// TODO: Add modernizr to CF so this works
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/05326ae435338ff88371acd83e8079c902759ed3